sw/device/rom_exts/rom_ext_manifest.S

```diff
@@ -38,22 +39,20 @@ image_version:
 image_timestamp:
   .dword 0x0
 
-// Overridden by signing script
-image_signature_algorithm:
-  .word 0x0
-
 // Overriden by signing script
-image_signature_exponent:
+image_signature_public_key_exponent:
   .word 0x0
 
-// Maybe: Overriden by signing script?
-image_usage_constraints:
+// reserved
   .word 0x0
 
-// Reserved
-  .word 0x0
+// Overriden by signing script?
+image_usage_constraints:
+  .rept 32
+  .byte 0x0
+  .endr
 
-// Maybe: Overriden by signing script?
+// Overriden by signing script?
```

Comment:
Is the question here - whether this information should be populated at the time of image creation or at the time of signing? The same goes for `image_usage_constraints`.

Reply:
Sorry, the question mark should go. This should be overridden by the signing script. The same goes for `usage_constraints.